package.json

```diff
@@ -2,7 +2,7 @@
   "name": "helloworld-cloud",
   "version": "0.2.0",
   "dependencies": {
-    "body-parser": "~1.0.2",
+    "body-parser": "~1.17.2",
```

Comment:
Updating body-parser to this version throws deprecate warnings when running grunt serve, so don't update it for now, or also update the code to avoid the deprecate warnings.

`GMT body-parser deprecated bodyParser: use individual json/urlencoded middlewares at lib/hello.js:8:13`
`GMT body-parser deprecated undefined extended: provide extended option at node_modules/body-parser/index.js:105:29`